packages/web-frontend/package.json

```diff
@@ -95,16 +94,7 @@
     "react-dev-utils": "^3.0.0",
     "react-error-overlay": "^1.0.7"
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+  "browserslist": [
+    "defaults"
```

Comment:
F some reason those settings cause an error with svg imports. Using the defaults works fine though and the list of browsers it includes is very similar so I am keen to go for the defaults. See here for a list of the browsers included in each. https://beyondessential.slack.com/archives/C01UPMP1LPN/p1630627300000900

Reply:
Thanks for that Tom! BTW I ran into the same issue in admin-panel, could you make the same change there as well? Thanks